plex_trakt_sync/plex_api.py

```diff
@@ -235,10 +300,17 @@ def __repr__(self):
         return "<%s:%s:%s>" % (self.guid.provider, self.guid.id, self.item)
 
     def to_json(self):
-        return {
+
+        metadata = {
             "collected_at": timestamp(self.collected_at),
             "media_type": "digital",
+            "resolution": self.resolution,
+            # "hdr": "dolby_vision",
```

Comment:
Why is this commented? Is there no way to make this work? 🤔

Reply:
I don't have any hdr movies and I can't see any hdr flag in the metadata of my own media. Could you provide the XML of an hdr movie?

Reply:
Where do I get this XML files from?

Reply:
Three dots on the movie poster —> Get Info —> View XML

Reply:
It seems that Trakt supports `dolby_vision`, `hdr10`, `hdr10_plus`, or `hlg`, here's some XML files:

* Dolby Vision: https://pastebin.com/q0tmT9cb
* HLG: https://pastebin.com/vDqKYkpc
* HDR / HDR+: https://pastebin.com/hWHtiNh2
	* I don't have samples to distinguish HDR / HDR+, but it would be nice to merge this only after we find what makes them different (I'll research it)

It looks to me that Dolby Vision can be identified by the presence of `DOVIPresent="1"`, HLG by the presence of `colorTrc="arib-std-b67"`, and HDR / HDR+ by the presence of `colorTrc="smpte2084"`. But I need to download more samples, because this is just my _best guess_.

Reply:
Found some samples for all formats, will post detailed results later 👍

Reply:
Here's a [gist](https://gist.github.com/rfgamaral/965257305c2fd386b9316fea3eff0f33) with XML files for all formats.

It looks exactly like my previous post, so, in a nutshell:

* For the **first** `MediaContainer.Video.Media.Stream` with `streamType=1` (video):
	* If `colorTrc="bt709"` ➔ Not HDR
	* If `colorTrc="smpte2084"` ➔ HDR/HDR+ (there's no way to distinguish them)
	* If `colorTrc="arib-std-b67"` ➔ HLG
	* If `DOVIPresent="1"` ➔ Dolby Vision

Reply:
Ok I've added those rules and tested with a sample video I found. You might wanna checkout this branch and test with a few of your movies like this:

```
> python main.py inspect 2468 

PlexTraktSync inspect [1b0b1bfb: hdr metadata support @2021-07-07 23:21:16]
Inspecting: <local:2468:<Movie:2468:Iphone11-4k-Recorder>>
Media.Guid: 'local://2468'
Media.Guids: []
Audio: 'None', 'Unknown (PCM Stereo)'
Video: 'hevc'
Guids:
  Guid: local://2468, Id: 2468, Provider: local
Metadata: {'collected_at': '2021-07-07:T21:41:46.000Z', 'media_type': 'digital', 'resolution': 'uhd_4k', 'hdr': 'hlg', 'audio': 'lpcm', 'audio_channels': '2.0'}
```

what you want to check that maps correctly is the `hdr` element in the `Metadata` row

replace 2468 with the `ratingKey` in your XML

Reply:
Works for me 👍

Reply:
@mcdallas ready for merge then?

Reply:
@glensc Yes i think its good to go